src/hw4_Polymorphism/Competition/Obstacle/Obstacle.java

```diff
@@ -0,0 +1,13 @@
+package hw4_Polymorphism.Competition.Obstacle;
+
+public abstract class Obstacle {
```

Comment:
why not interface?

Reply:
Because there are no methods to implemet. I also thought that abstact class would be better than a class because we shouldn't create an instances of Obstacle. But anyway, please explain why the interface is better, I want to be well in this topic! Thank you